### PR TITLE
fix multiselect

### LINKED ISSUE #5970: Weird checkboxes bug in Documents/Photos
Hi, 
There is a bug relative to file multi-selection mode in "Document" and "Photos" File list for Nextcloud 18. Due to the "folder description" on the top, there is a bug during this, I let you see the gif.

qjNlgIOcVH

I thought the UI revamp (https://github.com/nextcloud/android/issues/5207) was responsible of this, but no, it's a more "distant" bug.

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Fix  #5970

Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### Testing 
Writing tests is very important. Please try to write some tests for your PR. 
If you need help, please do not hesitate to ask in this PR for help.

[unit tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#unit-tests)
[instrumented tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#instrumented-tests)
[UI tests](https://github.com/nextcloud/android/blob/master/CONTRIBUTING.md#ui-tests)

- [x] Tests written, or not not needed

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
@Shagequi fancy taking a look? Works fine for me btw. :)

**Comment by Shagequi** [comment]
I watch for this ASAP :)

File: src/main/java/com/owncloud/android/ui/adapter/OCFileListAdapter.java [hunk 1]
```diff
@@ -201,7 +201,13 @@ public void removeAllFilesFromCheckedFiles() {
     }
 
     public int getItemPosition(OCFile file) {
-        return mFiles.indexOf(file);
+        int position = mFiles.indexOf(file);
+
+        if (shouldShowHeader()) {
+            position = position + 1;
```
Comment (Shagequi) [inline src/main/java/com/owncloud/android/ui/adapter/OCFileListAdapter.java:207]: The +1 is a[ magic number/value](https://en.wikipedia.org/wiki/Magic_number_(programming)), but it's okay :D

**Pull request merged**